/**:
  ros__parameters:

    model_size: pico 
    backbone_stage: 3
    neck_stage: 3
    model_path: .pt
    max_range: 30.0
    input_dim: 1081
    output_dim: 2